src/view/widget/tags.jsx

```diff
@@ -62,13 +62,12 @@ class Tags extends Component {
  *         _p: function() {...}
  *     }}
  *     tags={{...}}
- *     orderBy="name"
- *     order={1}
- *     amount={100}
- *     showCount={true} />
+ *     widget={{ orderBy: "name", order: 1, amount: 100, showCount: true }}
```

Comment:
Better to write this configuration example in multiple lines with proper indentation for better readability,
```
widget={{
  order_by: "name",
  amount: 100,
  show_count: true
}}
```